pkg/pvtdatastorage/cdbpvtdatastore/test_exports.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
```

Comment:
Is it correct that SecureKey isn't also listed? Did we make any non-trivial modifications to this file?

Reply:
fixed in the next pr